package.json

```diff
@@ -42,5 +42,5 @@
     "trailingComma": "none",
     "arrowParens": "avoid"
   },
-  "packageManager": "pnpm@6.27.1"
+  "packageManager": "pnpm@7.3.0"
```

Comment:
I guess the previous version is incorrect, as on `v6.27.1` I received warning:
```
Your pnpm-lock.yaml was generated by a newer version of pnpm. It is a compatible version but it might get downgraded to version 5.3
```